cli-feed-v4.json

```diff
@@ -46,7 +46,7 @@
       "hidden": false
     },
     "v4-prerelease": {
-      "release": "4.86.0",
+      "release": "4.87.0",
```

Comment:
Can you include a diff gist in the PR description?

Reply:
@soninaren  As requested please find the Gist added here 
https://gist.github.com/v-imohammad/2eafa0aeebde12ae78b8373e0416af91/revisions